myth/CIModules/auth/models/Login_model.php

```diff
@@ -128,8 +128,13 @@ public function distributedBruteForceTime()
 
             // Compute our daily average over the last 3 months.
             $avg_start_time = date('Y-m-d 00:00:00', strtotime('-3 months'));
-
-            $query = $this->db->query("SELECT COUNT(*) / COUNT(DISTINCT DATE(`datetime`)) as num_rows FROM `auth_login_attempts` WHERE `datetime` >= ?", $avg_start_time);
+            $selected_fields = array("COUNT(*) / COUNT(DISTINCT DATE(`datetime`)) num_rows");
+	        $tablename = "auth_login_attempts";
```

Comment:
yes the main problem was executing query directly and without query builder.
I have found this problem when i set a table prefix and found that this part do not add this prefix.